frontend/src/views/housing/enquiry/EnquiryIntakeView.vue

```diff
@@ -30,5 +32,6 @@ onMounted(async () => {
     v-if="!loading"
     :activity-id="props.activityId"
     :enquiry-id="props.enquiryId"
+    :assigned-activity-id="props.assignedActivityId"
```

Comment:
Do you think we should pull that confirmation bit out into its own component instead of having the form pulling double duty? And have the view dictate what is displayed? Same for the SubmissionIntakeView